web/profiles/joinup/src/Plugin/Block/AddContentBlock.php

```diff
@@ -18,12 +22,55 @@
  *  admin_label = @Translation("Add content"),
  * )
  */
-class AddContentBlock extends BlockBase {
+class AddContentBlock extends BlockBase
+{
```

Comment:
This is not correct according to the coding standards. We definitely need to look at the automated coding standards check. Something seems to be wrong there.